core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -500,6 +505,28 @@ protected Node(final Environment environment, Collection<Class<? extends Plugin>
         }
     }
 
+    static void checkForIndexDataInDefaultPathData(final NodeEnvironment nodeEnv, final Logger logger) throws IOException {
```

Comment:
Sorry, I'm struggling to understand this comment? There is nothing being returned here? Help?